### PR TITLE
Resolve ticket #4.

### PR DESCRIPTION
**Pull request by mwoguri**
Added ice cream to list. #4

### CODE REVIEW

File: shoppingList.txt [hunk 1]
```diff
@@ -7,3 +7,4 @@ Chiefs jersey
 Flannel shirt from costco
 Beefjerky
 More beefjerky!
+Icecream
```
Comment (adamkendis) [inline shoppingList.txt]: This is a comment.

File: shoppingList.txt [hunk 2]
```diff
@@ -7,3 +7,4 @@ Chiefs jersey
 Flannel shirt from costco
 Beefjerky
```
Comment (adamkendis) [inline shoppingList.txt]: This is another comment.

File: shoppingList.txt (hunk 1 above)
Comment (mwoguri) [reply shoppingList.txt]: Good catch, thanks!

File: shoppingList.txt (hunk 2 above)
Comment (mwoguri) [reply shoppingList.txt]: This is because XYZ.

**Pull request merged**